framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -1003,6 +1003,9 @@ protected boolean visitWildcardSupertype(
 
     protected boolean visitWildcardSubtype(
             AnnotatedWildcardType subtype, AnnotatedTypeMirror supertype, VisitHistory visited) {
+        if (subtype.isUninferredTypeArgument()) {
+            return ignoreUninferredTypeArguments;
```

Comment:
For the first missing error "Issue329.java:37" I'm fine with adding a reference to Issue #979 and not having the error.

The second missing error is on "MethodTypeVars.java:14"; is this referring to https://github.com/typetools/checker-framework/blob/master/checker/tests/nullness/generics/MethodTypeVars.java#L14 ? Why is that test affected? It's a static method call with null as argument. I think we specifically ignore null as arguments, which will also result in un-inferred type arguments here. Do you think we can easily special case this?

I like the idea of outputting a message if we couldn't infer some type arguments! Should it be a warning for every time we failed doing this? Or should it be a single notice for the whole compilation, with an option to issue a warning for every instance (similar to how unchecked warnings work)? The latter would allow us to work even with -werror.
Maybe if `-AconservativeUninferredTypeArguments` it is a warning for each instance and without `-AconservativeUninferredTypeArguments` it is a single notice?